registry/remote/registry_example_test.go

```diff
@@ -84,7 +84,7 @@ func ExampleRegistry_Repositories() {
 }
 
 // ExampleRegistry_Repositories_second gives example snippets for listing respositories in a HTTP registry with pagination.
-func ExampleRegistry_Repositories_second() {
+func ExampleRegistry_Repositories_http() {
```

Comment:
@shizhMSFT do we even need the non secure example? 
I think the flag is public on the repository so the docs on the flag are probably good enough. This is the same flow with just a Boolean toggle. So didn’t seem it’s necessary.

Reply:
@qweeah Let's remove this example and just keep the default HTTPS one.